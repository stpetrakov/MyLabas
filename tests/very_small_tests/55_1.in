32307 22565 31736 28468 24147 18352 29662 20784 921 32113 19482 7487 23910 350 10908 26656 22988 17484 19379 4214 25144 14853 980 18806 2900 22938 3735 15843 31787 30715 20226 24380 20623 12484 25866 13620 31415 20826 15447 16738 26669 31287 30792 19258 28136 27266 24887 10300 16308 10723 14078 21611 30427 1173 3257 