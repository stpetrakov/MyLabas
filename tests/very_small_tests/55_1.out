350 921 980 1173 2900 3257 3735 4214 7487 10300 10723 10908 12484 13620 14078 14853 15447 15843 16308 16738 17484 18352 18806 19258 19379 19482 20226 20623 20784 20826 21611 22565 22938 22988 23910 24147 24380 24887 25144 25866 26656 26669 27266 28136 28468 29662 30427 30715 30792 31287 31415 31736 31787 32113 32307 